documentation/dev-portal/src/faq/integrations-faq.md

```diff
@@ -198,7 +198,7 @@ For the moment then yes, the mixnet is free to use. There are no limits on the a
 No, although we do recommend that apps that wish to integrate look into running some of their own infrastructure such as gateways in order to assure uptime. 
 
 ### How can I find out if an application is already supported by network requester services? 
-You can check the [default allowed list](https://nymtech.net/.wellknown/network-requester/standard-allowed-list.txt) file to see which application traffic is whitelisted by default. If the domain is present on that list, it means that existing [network requesters](https://nymtech.net/docs/nodes/network-requester.html) can be used to privacy-protect your application traffic. Simply use [NymConnect](../quickstart/nymconnect-gui.md) to connect to this service through the mixnet.
+You can check the [default allowed list](https://nymtech.net/.wellknown/network-requester/standard-allowed-list.txt) file to see which application traffic is whitelisted by default. If the domain is present on that list, it means that existing [network requesters](https://nymtech.net/docs/nodes/network-requester.html) can be used to privacy-protect your application traffic. 
```

Comment:
I am not sure if this will not be taken away for good soon and only the exit policy will remain. Maybe let's double check before publishing this as truth source.

Reply:
yeah this needs to be removed, just didnt get around to text modification yet, wanted to have your opinion on the new structure first :)